docker-compose.yml

```
@@ -14,6 +14,8 @@ services:
     restart: always
     container_name: drivers
     hostname: drivers
+    environment:
+      - CRYSTAL_PATH=/lib/local-shards
```

```suggestion
      - CRYSTAL_PATH=lib:/lib/local-shards
```
[Default behaviour](https://github.com/crystal-lang/crystal/blob/c41640fc205840a544f23eda33a6fedb4a184088/src/compiler/crystal/crystal_path.cr#L23) is to preference default path over custom unless specified explicitly in the ordering. In this case it's preferable to let users update/add shards externally and to run tests without container rebuilds.